### ISSUE TITLE
LiteDB.Studio for earlier versions of LiteDB

### ISSUE THREAD

**Issue by izlotnik** [mbdavid/LiteDB.Studio#4]
It would be great to be able to work with earlier versions of LiteDB. We are still using LiteDB v3.0 and we are not able to read (without conversion) or write our database files. Any plans to support earlier releases of LiteDB?

**Comment by mbdavid** [comment]
Yes, I have plans to LiteDB v5 migrate from earlier versions... but will be a migration. After open datafile in LiteDB Studio, you datafile will be converted into new format.

**Comment by izlotnik** [comment]
That is a great feature to be able to open LiteDB files created in the older releases! At the same time, I would like to be able to update the LiteDB file and store it in the same version as it was before, e.g. v4, because the software I use does not understand the format used in v5. Do you have any plans to support this workflow?

**Comment by RichardClaxton** [comment]
It was Version 1.0.1 

I might try some older versions.
